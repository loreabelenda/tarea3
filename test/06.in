12 test/Ejercicio1_cortado.dat
