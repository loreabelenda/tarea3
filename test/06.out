Ejercicio 1
(Kleinberg &
Tardos, Ex.
